mdaviz/select_fields_table_view.py

```diff
@@ -20,6 +20,8 @@
 from .select_fields_table_model import TableColumn
 from .select_fields_table_model import TableField
 
+HEADERS = "Field", "X", "Y", "I0", "Unscale", "PV", "DESC", "Unit"
```

Comment:
Would `"Monitor"` be a more general term here?  `"I0"` pulls my attention to a specific `I0` signal when counting time, I00, I000, or some other signal might be selected.

Reply:
Done